src/test/java/tictactoe/core/HumanPlayerTest.java

```diff
@@ -23,14 +31,16 @@ public void playerSymbol() {
 
     @Test
     public void chooseMove() {
-        Player human = new HumanPlayer(PlayerSymbol.X, new MockUI(3));
+        Player human = new HumanPlayer(PlayerSymbol.X, mockMediator);
         Board board = new Board(3);
 
+        human.requestMove(board);
         assertEquals(
-                "human can request a move from the UI",
-                3,
-                human.chooseNextMove(board)
+                "human can request a move via the mockMediator",
+                "request move",
+                mockMediator.getLog()
         );
     }
+
```

Comment:
Is this extra space on purpose? Am I not quite sure it's consistent with your file

That's a pick detail though